AElf.Kernel.Core.Tests/KernelCoreTestAElfModule.cs

```diff
@@ -69,6 +71,11 @@ public override void ConfigureServices(ServiceConfigurationContext context)
                    
                     return dataProvider.Object;
                 });
+            
+            //For BlockchainStateMergingService testing
```

Comment:
remove

Reply:
@loning thx boss for code review, learned a lot~